conda-recipes/llvmdev/conda_build_config.yaml

```
@@ -1,17 +1,14 @@
 # Numba/llvmlite stack needs an older compiler for backwards compatability.
 c_compiler_version:         # [linux or osx]
-  - 7                       # [linux and (x86_64 or ppc64le)]
-  - 11                      # [linux and aarch64]
+  - 11                      # [linux]
   - 14                      # [osx]
 
 cxx_compiler_version:       # [linux or osx]
-  - 7                       # [linux and (x86_64 or ppc64le)]
-  - 11                      # [linux and aarch64]
+  - 11                      # [linux]
   - 14                      # [osx]
 
-fortran_compiler_version:   # [linux]
-  - 7                       # [linux and (x86_64 or ppc64le)]
-  - 11                      # [linux and aarch64]
+fortran_compiler_version:   # [linux or osx]
```

```suggestion
fortran_compiler_version:   # [linux]
```
I don't think there's any need for the `or osx` as there's no subsequent selection available?